src/components/Error/Empty.jsx

```diff
@@ -4,8 +4,9 @@ import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 
 const Empty = translate()(({ t, type, canUpload, localeKey }) => {
+  // The role="main" is here to get a blur effect on mobile when loading
```

Comment:
That is a very good usage for comments. I asked myself what's the need of adding `role="main"` to the div ;-)
I don't catch the difference between `<main />` and `<div role="main" />` but that's another story I'll study myself later on.